website/content/SDK/keys/_index.md

```diff
@@ -1,6 +1,64 @@
 ---
-title: keys
-keywords: ""
-excerpt: ""
+title: API Keys
+keywords: "api,keys,key,secret,token,bearer,credentials,secrets,data,sources"
+excerpt: "This guide explains how to manage API keys within an OpenBB SDK environment."
 geekdocCollapseSection: true
 ---
+The `keys` module provides the same functionality as the <a href="https://openbb-finance.github.io/OpenBBTerminal/#accessing-other-sources-of-data-via-api-keys" target="_blank"> Keys menu</a> does in the <a href="https://openbb-finance.github.io/OpenBBTerminal/terminal/" target="_blank">OpenBB Terminal application</a>. Each data source requiring authentication will have slight differences of required inputs. Refer to the table <a href="https://openbb-finance.github.io/OpenBBTerminal/#accessing-other-sources-of-data-via-api-keys" target="_blank">here</a> for a compiled list of links for each data source.
```

Comment:
Rename "OpenBB Terminal application" to "OpenBB Terminal" (that's how we call it everywhere) and change the link to https://openbb.co/products/terminal.